F1WM/Repositories/EnginesRepository.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using System.Threading.Tasks;
+using F1WM.ApiModel.Engines;
+using F1WM.DatabaseModel;
+using Microsoft.EntityFrameworkCore;
+
+namespace F1WM.Repositories
+{
+	public class EnginesRepository: RepositoryBase, IEnginesRepository
+	{
+		public async Task<Engines> GetEngines(char letter)
+		{
+			Engines result = new Engines();
+
+			 result.EnginesList = 
```

Comment:
The formatting is weird. Anyway, it's going to change once you use proper DB->API mapping. This is a perfect place to use ProjectTo as well.